docs/guide-end-of-form.rst

```diff
@@ -185,23 +205,41 @@ In many workflows, it's important to guarantee that data is not changed after th
 
 When data collectors reach the form end screen, they only have the option to :guilabel:`Finalize`. If they are interrupted during a form filling session, they need to exit and discard changes or rely on automatic data backups and recovery (the partially-filled form will open automatically when they open the same blank form again).
 
-Edits may be needed to some drafts
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-In some workflows, new information may need to be added to a form after a first data collection event. For example:
+.. note::
+  If you would still like to leave the door open to edits but they are generally strongly discouraged, you could leave the option to save as draft from the form filling experience and use :ref:`audit logging <guide-form-states-form-audit-log>` with change tracking and reasons for change to get more information on edits that are made.
+
+Edits are needed to specific, known drafts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Saving as draft and making edits can be an expected and important part of a workflow. For example:
 
+* some specific filled forms may require adding details that were initially unavailable.
 * a form may capture data from multiple days.
-* a data collection subject may be observable at different times or from different angles, revealing new information.
-* review to fix small mistakes like typos may be encouraged.
 * tasks like transcribing an audio recording may be needed.
 
+This can generally be addressed in form design by using required questions. You can also use a ``note``, a ``hint``, or a ``required_message`` to prompt users to exit the form and :guilabel:`Save as draft` at known points of the form.
+
+If you have designed your form so that it's only possible to reach the form end screen once all necessary tasks are complete, you may want to :ref:`hide the Save as draft button from the form end screen <guide-form-states-hide-save-draft-form-end>`. It can be easier for data collectors to take the right action if they always use the back button to save as draft and only use the form end screen to finalize/send.
+
+Edits are needed but it's not known ahead of time to which drafts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In other workflows, edits are needed but it is not known ahead of time which drafts will be edited. For example:
```

Comment:
"In other workflows, it is not known ahead of time which drafts will be edited." So it's not the same as heading